openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js

```diff
@@ -0,0 +1,234 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import {
+    addOwner,
+    addTag,
+    addTier,
+    checkAddGroupWithOperator,
+    checkAddRuleWithOperator,
+    checkmustPaths,
+    checkmust_notPaths,
+    CONDITIONS_MUST,
+    CONDITIONS_MUST_NOT,
+    FIELDS,
+    OPERATOR
+} from '../../common/advancedSearch';
+
+import {
+    interceptURL, mySqlConnectionInput, testServiceCreationAndIngestion, verifyResponseStatusCode
+} from '../../common/common';
+
+import { API_SERVICE } from '../../constants/constants';
+
+import { MYSQL } from '../../constants/service.constants';
+
+const service_name = MYSQL.serviceName;
+
+describe('Advance search should work properly', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.get('[data-testid="appbar-item-explore"]').and('be.visible').click();
+  });
+
+  it('Pre-requisite for advance search', () => {
+    addOwner(FIELDS.Owner.searchCriteriaFirstGroup);
+    addTier(FIELDS.Tiers.searchCriteriaFirstGroup);
+    addTag(FIELDS.Tags.searchCriteriaFirstGroup);
+  });
+
+  it('Mysql ingestion', () => {
+    interceptURL(
+      'GET',
+      'api/v1/teams/name/Organization?fields=*',
+      'getSettingsPage'
+    );
+    cy.get('[data-testid="appbar-item-settings"]').should('be.visible').click();
```

Comment:
why we are Ingesting the mysql?

Reply:
We only have one service, database and database schema and for Add-group and add-rule we need to have second one for comparison